video2dataset/main.py

```diff
@@ -59,10 +62,25 @@ def video2dataset(
     optical_flow_fps: int = -1,
     optical_flow_downsample_size: int = None,
     optical_flow_dtype: type = np.float16,
+    sampler = lambda x: x,
+    slurm_cpus_per_task: int=1,
```

Comment:
not sure if this is the best way to do these args

Reply:
what would you propose instead?